staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go

```diff
@@ -215,12 +215,13 @@ func captureStdErr() (func() string, func(), error) {
 		<-readerClosedCh
 		return buf.String()
 	}
-	klog.LogToStderr(true)
```

Comment:
I don't think we need to change this line.

I think we only need to change the order of `os.Stderr=stderr` and `klog.LogToStderr(false)` in the cleanup function.

Reply:
``` diff
-	klog.LogToStderr(true)
	cleanUp := func() {
+		klog.LogToStderr(false)
		os.Stderr = stderr
-		klog.LogToStderr(false)
		stopReadingStdErr()
	}

+	klog.LogToStderr(false)
	os.Stderr = writer
	go func() {
		io.Copy(&buf, reader)
		readerClosedCh <- struct{}{}
		close(readerClosedCh)
	}()
	klog.LogToStderr(true)
```

I think `klog.LogToStderr(false)` must have before every `os.Stderr` modification.
There is also an `klog.LogToStderr(true)` at the bottom of this function to reenable Log